prog/minimization_test.go

```diff
@@ -139,6 +139,46 @@ func TestMinimize(t *testing.T) {
 	}
 }
 
+func TestMinimizeTest(t *testing.T) {
+	tests := []struct {
+		orig            string
+		callIndex       int
+		pred            func(*Prog, int) bool
+		result          string
+		resultCallIndex int
+	}{
+		// Make sure we don't hang when minimizing resources.
+		{
+			"r0 = test$res0()\n" +
+				"test$res1(r0)\n",
+			-1,
+			func(p *Prog, callIndex int) bool {
+				return false
+			},
+			"r0 = test$res0()\n" +
+				"test$res1(r0)\n",
+			-1,
+		},
+	}
+	target, _, _ := initRandomTargetTest(t, "test", "64")
+	for ti, test := range tests {
```

Comment:
This is total copy-paste of TestMinimizeLinux? Please deduplicate code, it should be easy to do.

Reply:
Done, PTAL